src/components/select.rs

```diff
@@ -93,6 +106,19 @@ where
     }
 
     fn change(&mut self, props: Self::Properties) -> ShouldRender {
+        if self.props.selected != props.selected {
+            if let Some(select) = self.select_ref.cast::<SelectElement>() {
+                let val = props
+                    .selected
+                    .as_ref()
+                    .map(|v| v.to_string())
+                    .unwrap_or_else(|| "".into());
```

Comment:
nit: you could use `unwrap_or_default()` here

Reply:
Thanks! Fixed it.